tests/ssl/CMakeLists.txt

```diff
@@ -2,13 +2,16 @@ cmake_minimum_required(VERSION 3.15)
 project (crow_ssl_test)
 
 include(${CMAKE_SOURCE_DIR}/cmake/compiler_options.cmake)
+include(${CMAKE_SOURCE_DIR}/cmake/CPM.cmake)
+
+CPMAddPackage("gh:catchorg/Catch2@3.6.0")
```

Comment:
Isn't that inclusion superfluous as it is also included one level above?

Reply:
Yes indeed it was.  Removed in next commit.

Reply:
MAddPackage("gh:catchorg/Catch2@3.6.0") was already done in parent dir /test, so it could be removed here too?

Reply:
Agreed and removed.